src/index.html

```diff
@@ -13,10 +13,94 @@
     <title>Moyo header</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          class="logo__img"
+          alt="Moyo"
+          src="images/logo.svg"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav_item">
+            <a
+              class="is-active nav__link"
+              href="index.html"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav_item">
```